roles/external_secrets_operator/vars/main.yml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+_external_secrets_operator_helm_values:
+  global:
+    nodeSelector: "{{ external_secrets_operator_node_selector }}"
+  image:
+    repository: "{{ external_secrets_operator_image | atmosphere.common.docker_image('name') }}"
+    tag: "{{ external_secrets_operator_image | atmosphere.common.docker_image('tag') }}"
+  installCRDs: true
+  leaderElect: true
+  certController:
+    create: false
+  webhook:
+    create: false
```

Comment:
Any commentary why those are disabled?

Reply:
Certcontroller is for webhook and webhook for integrating secretstores/clustersecretstores with custom backends or for something like secret field masking. We are not planning to use nothing custom. Never found any of this beneficial. Having secretstore/clustersecretstore with backend ACL binding to it is enaught.